syntax = "proto3";

package SMSC.Protobuf.SMPP;

import "Definition.proto";

message Deliver_Sm_Req {
  string smsc_unique_id = 1;

  SMPP_SM smpp = 2;
  Message_Body body = 3;

  string source_client_id = 4;
}

message Deliver_Sm_Resp {
  string smsc_unique_id = 1;
  
  int32 error_code = 2;
}
